include/EffectView.h

```diff
@@ -62,7 +65,7 @@ class EffectView : public PluginView
 	}
 
 	static constexpr int DEFAULT_WIDTH = 215;
-	static constexpr int DEFAULT_HEIGHT = 60;
+	static constexpr int DEFAULT_HEIGHT = 47;
```

Comment:
```suggestion
	static constexpr int DEFAULT_HEIGHT = 44;
```

From my mind calculation, 44 seems to be a better default height. Try using it and see.

Reply:
Nvm then, 47 is enough. It's already better in the current state.